reactance/reactance.md

```diff
@@ -0,0 +1,75 @@
+# Reactance in an RLC Circuit
+The opposition to a flow of alternating current in a circuit that arises due to
+the presence of inductors and capacitors. Though it only occurs in AC circuits,
+it is analogous to resistance in DC circuits.
+
+Reactance causes the voltage and current of the circuit to be out of phase with
+each other. The type and magnitude of reactance determines whether current
+leads or lags behind the voltage.
+
+### Formula for Total Reactance
+X = X_L - X_C , where X_L is the inductive reactance and X_C is the capacitive reactance.
+
+A positive X value indicates that the circuit's inductive reactance dominates
+the capacitive reactance and so the current will lag behind the voltage.
+
+A negative X value indicates that the circuits capacitive reactance dominates
+the inductive reactance and so the current leads the voltage.
+
+
+## Inductive Reactance
+Inductive reactance is the opposition to an alternating current caused by an
+inductor.
+
+Inductors resist changes in current by creating a magnetic which stores energy
+and opposes the flow of current via induction. In a purely inductive circuit
+the current lags behind the voltage by 90 degrees.
+
+### Inductive Reactance: X_L
+X_L = (2)(pi)(*f*)(L) , where X_L is measured in ohms, *f* is the frequency
```

Comment:
do you want the space here?

Reply:
Not sure if you mean the newline or the empty space between the "###" and "Inductive".

The newline is there because I placed an extra line between all of the H2 sections.  The empty space between ### and Inductive is there so that the markdown parsers recognizes that it should be rendered as <h3> text and not include the 3 # signs.